exercises/scale-generator/scale_generator_test.exs

```diff
@@ -0,0 +1,94 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("scale_generator.exs", __DIR__)
+end
+
+ExUnit.start
+ExUnit.configure exclude: :pending, trace: true
+
+defmodule ScaleGeneratorTest do
+  use ExUnit.Case
+
+  @major_scale_pattern           "MMmMMMm"
+  @minor_scale_pattern           "MmMMmMM"
+  @dorian_scale_pattern          "MmMMMmM"
+  @mixolydian_scale_pattern      "MMmMMmM"
+  @lydian_scale_pattern          "MMMmMMm"
+  @phrygian_scale_pattern        "mMMMmMM"
+  @locrian_scale_pattern         "mMMmMMM"
+  @harmonic_minor_scale_pattern  "MmMMmAm"
+  @octatonic_scale_pattern       "MmMmMmMm"
+  @hexatonic_scale_pattern       "MMMMMM"
+  @pentatonic_scale_pattern      "MMAMA"
+  @enigmatic_scale_pattern       "mAMMMmm"
+
+  #@tag :pending
+  test "C Major scale" do
+    assert ScaleGenerator.scale("C", @major_scale_pattern) == ~w(C D E F G A B)
```

Comment:
I know you copied this from other tracks. However, I feel that these tests force the user to go from 0 to 60. One of the goals of an exercism exercise is to re-create the TDD experience by carefully building up behavior in each subsequent test. Are there some intermediate tests we could add to help folks get to the solution more iteratively?